Makefile

```
@@ -145,6 +145,12 @@ install-models-hub-torch:
 test-torch:
 	pytest ${COVERAGE_ARGS} tests/torch -m "not weekly and not nightly and not models_hub" --junitxml ${JUNITXML_PATH} $(DATA_ARG)
 
+test-torch-cpu:
+	pytest ${COVERAGE_ARGS} tests/torch -rS -m "not cuda and not weekly and not nightly and not models_hub" --junitxml ${JUNITXML_PATH} $(DATA_ARG)
```

let's try run them in parallel
```suggestion
	pytest -n logical ${COVERAGE_ARGS} tests/torch -rS -m "not cuda and not weekly and not nightly and not models_hub" --junitxml ${JUNITXML_PATH} $(DATA_ARG)
```